docs/docs/starlark-reference/plan.md

```diff
@@ -221,6 +221,17 @@ render_templates
 
 The `render_templates` instruction combines a template and data to produce a [files artifact][files-artifacts-reference]. Files artifacts can be used with the `files` property of the `ServiceConfig` object, allowing for reuse of config files across services.
 
+**Returns**: a [future reference][future-references-reference] resolving to a `string` representing the name of a [files artifact][files-artifacts-reference].
```

Comment:
Here's link to Google docstring format, if you're interested: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#:~:text=def%20function_with_types_in_docstring(param1%2C%20param2)%3A